mocks/flask-react-mocks/server/server.py

```diff
@@ -0,0 +1,54 @@
+from flask import Flask
+from flask import jsonify
+from flask import request
+from flask import make_response
+from sampleData import *
+from constants import CONSTANTS
+import ast
+import datetime
+app = Flask(__name__)
+
+# List Endpoints
+@app.route(CONSTANTS['ENDPOINT']['LIST'])
+def getList():
+    return jsonify(
+        sampleData['listTextAssets']
+    )
+
+@app.route(CONSTANTS['ENDPOINT']['LIST'], methods = ['POST'])
+def addListItem():
+    data = request.data.decode('utf-8')
+    data = ast.literal_eval(data)
+    listItem = {'_id': sampleData['listId'], 'text': data['text']}
+    sampleData['listTextAssets'].append(listItem)
+    sampleData['listId'] += 1
+    return jsonify(
+        listItem
+    )
+
+@app.route(CONSTANTS['ENDPOINT']['LIST'] + '/<int:id>', methods=['DELETE'])
+def deleteListItem(id):
+    listItemToRemove = [listItem for listItem in sampleData['listTextAssets'] if listItem['_id'] == id]
```

Comment:
We know that the id is unique so why create a list? It also seems that this isn't used anywhere but for checking if a list item can be removed.

Maybe we should try something like this?
```py
for i in range(len(sampleData['listTextAssets'])): 
    if sampleData['listTextAssets'][i]['_id'] == id: 
        del sampleData['listTextAssets'][i] 
        return jsonify(....)
return jsonify(error here)
```

Reply:
Unless you have a reason for this

Reply:
Agreed!